CHANGELOG.md

```diff
@@ -1,4 +1,68 @@
 
+<!-- 8.0.0-beta3 START -->
+
+# 8.0.0-beta3 (2021-06-01)
+
+### Features and enhancements
+
+* ** CloudWatch:** Add Lambda@Edge Amazon CloudFront metrics. [#34561](https://github.com/grafana/grafana/pull/34561), [@razor-x](https://github.com/razor-x)
+* ** Time series panel:** Position tooltip correctly when window scrolled or resized. [#34782](https://github.com/grafana/grafana/pull/34782), [@dprokop](https://github.com/dprokop)
```

Comment:
Nit: These two need a PR rename after this is merged I think

Reply:
@hugohaggmark updated both of these

Reply:
Thank you!